frontend/packages/pipelines-plugin/src/plugin.tsx

```diff
@@ -69,310 +20,6 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.default),
     },
   },
```

Comment:
Not sure if the overview section extension is being used anymore with the new topology sidebar extensions. @debsmita1 Can check if this is still being used?